scripts/build_linux.py

```diff
@@ -89,15 +89,14 @@ def gngen(arch, ssl_root, msdk_root, quic_root, scheme, tests, use_gcc, fake_aud
         gn_args.append('owt_include_tests=false')
     if use_gcc:
         gn_args.extend(['is_clang=false', 'use_lld=false', 'use_sysroot=false', 'use_custom_libcxx=false', 'treat_warnings_as_errors=false'])
-    if fake_audio:
-        gn_args.extend(['rtc_include_pulse_audio=false', 'rtc_include_internal_audio_device=false'])
     if shared:
         gn_args.extend(['rtc_enable_protobuf=false', 'is_component_build=true'])
     else:
         gn_args.extend(['is_component_build=false'])
     ffmpeg_branding_name = "Chrome"
     if cloud_gaming:
         ffmpeg_branding_name = "OWT"
+        gn_args.extend(['rtc_include_pulse_audio=false', 'rtc_include_internal_audio_device=false'])
```

Comment:
We were building closed source OWT which we attempt to upstream with these flags:
```
  python scripts/build_linux.py --gn_gen --sdk --output_path dist \
    --scheme release --arch x64 --shared
```
Moreover, `--fake_audio` flag was explicitly removed with the change:
```
commit 2cc0e8d4559a5024a495a9b0156031bb02301661
Author: akodanka <anoob.anto.kodankandath@intel.com>
Date:   Thu Apr 15 21:28:41 2021 +0530

    Enable Pulseaudio to list audio devices in containers

    Enable audio in OWT library, which is required for Audio devices.
    Included dev packages(libasound2-dev, libpulse-dev) in build image.
    Included pulseaudio package and config in the final image.

    Signed-off-by: akodanka <anoob.anto.kodankandath@intel.com>
```

So, we need discussion whether we can tight  fake_audio to cloud gaming build. As of now we do have evidence that we can't and we will need to fix OWT initialization issue I see with upstream OWT.